### ISSUE TITLE
Validate ECP5 pinout

### ISSUE THREAD

**Issue by oskirby** [oskirby/logicbone#1]
The caBGA381 is a big package, with a lot of balls. That's a lot of data entry that is ripe for user error and we should take some time to review the pinout to make sure its correct before manufacturing begins.

We also need to pay attention to compatibility between logic sizes, and the ECP5-5G variants to get an idea of which parts can be substituted.

**Comment by oskirby** [comment]
It seems that the ECP5 and ECP5-5G versions are interchangeable with one another, with a few tweaks to voltage programming (1.2V core and SERDES instead of 1.1V).

On the 45F logic sizes, pads K16 and K17 are no-connect, and fortunately we are not using them for anything on this design (they were sacrificed to make room for more bypass caps).

TLDR: This design is compatible with both the ECP5 and ECP5-5G in either the 45F or 85F logic sizes.
